packages/vuetify/src/styles/objects/_index.sass

```diff
@@ -16,28 +16,23 @@
       color: map-deep-get($material, 'text', 'disabled') !important
 
 .v-application
+  backface-visibility: hidden
   display: flex
+  display: flex
+  flex-direction: column
+  flex: 1 1 auto
+  max-width: 100%
+  min-height: 100vh
+  position: relative
 
   a
     cursor: pointer
 
   &--is-rtl
     direction: rtl
 
-  &--wrap
```

Comment:
Are we dropping IE then?

Reply:
I assume the wrapper is needed for flex in ie?

Reply:
#2328, #2298